cmd/oras/login.go

```diff
@@ -38,13 +38,14 @@ type loginOptions struct {
 	hostname  string
 	fromStdin bool
 
-	debug     bool
-	configs   []string
-	username  string
-	password  string
-	insecure  bool
-	plainHttp bool
-	verbose   bool
+	debug      bool
+	configs    []string
+	caFilePath string
+	username   string
+	password   string
+	insecure   bool
+	plainHttp  bool
+	verbose    bool
```

Comment:
It seems that those options are common for all operations like `oras login/pull/push/copy/discover`. Could we refactor it to `commonOptions` in `common.go`?

Reply:
The refactor effort can be in another PR.

Reply:
Created an issue for it https://github.com/oras-project/oras/issues/385